clarity/src/vm/docs/mod.rs

```diff
@@ -1912,8 +1912,8 @@ const STX_GET_BALANCE: SimpleFunctionAPI = SimpleFunctionAPI {
     signature: "(stx-get-balance owner)",
     description: "`stx-get-balance` is used to query the STX balance of the `owner` principal.
 
-This function returns the STX balance of the `owner` principal. In the event that the `owner`
-principal isn't materialized, it returns 0.
+This function returns the STX balance, in microstacks (1 STX = 1000000 microstacks), of the
```

Comment:
Can you write 1000000 as 1,000,000 so it's clear that it's 1 million?  Thanks!

Reply:
Addressed in 2bb0ee182.